internal/impl/postgresql/pglogicalstream/sanitize/sanitize.go

```diff
@@ -366,6 +366,18 @@ func SQLQuery(sql string, args ...any) (string, error) {
 // ValidatePostgresIdentifier checks if a string is a valid PostgreSQL identifier
 // This follows PostgreSQL's standard naming rules
 func ValidatePostgresIdentifier(name string) error {
+	if parts := strings.Split(name, "."); len(parts) == 2 {
```

Comment:
Incorrect if the identifier is quoted. Also not sure of the logic here, if a schema is passed won’t you end up with `schema.schema.table`?

Reply:
This is gone now